infrastructure/ansible/roles/dataset_loader/defaults/main.yml

```diff
@@ -1297,6 +1325,39 @@ dl_ds_default_profile_layers:
         - name: LOCAL proxy.local.cluster.type
           configFile: records.config
           value: INT 3
+        # Param value changed between ATS 7 & 8
+        - name: CONFIG proxy.config.net.sock_option_flag_in
+          configFile: records.config
+          value: INT 5
+        # ATS 8 specific param
+        - name: CONFIG proxy.config.net.max_connections_active_in
+          configFile: records.config
+          value: INT 100000
+        # ATS 8 specific param
+        - name: CONFIG proxy.config.http.origin_max_connections_queue
+          configFile: records.config
+          value: INT 0
+        # ATS 8 specific param
+        - name: CONFIG proxy.config.http.parent_proxy_routing_enable
+          configFile: records.config
+          value: INT 1
+      v9:
+        # Param change for ATS 9, from proxy.config.net.max_connections_active_in
+        - name: CONFIG proxy.config.net.max_requests_in
+          configFile: records.config
+          value: INT 100000
+        # Param change for ATS 9, from proxy.config.http.origin_max_connections_queue
+        - name: CONFIG proxy.config.http.per_server.connection.queue_size
+          configFile: records.config
+          value: INT 0
+        # Param value changed between ATS 7,9
+        - name: CONFIG proxy.config.net.sock_option_flag_in
+          configFile: records.config
+          value: INT 1
+        # Param value changed between ATS 8 & 9, default is 0
+        - name: CONFIG proxy.config.ssl.handshake_timeout_in
+          configFile: records.config
+          value: INT 30
```

Comment:
Is this to say that the default changed between 8 & 9 and we're preserving the older value of 30 in ATS 9?

Reply:
This parameter was removed from the ATS 9-specfiic section. 

`proxy.config.ssl.handshake_timeout_in` has a default value of 0 in ATS 7 and 8, and a default value of 30 in ATS 9. So, no sense in keeping it in `dataset_loader`.